arviz/stats/stats.py

```diff
@@ -629,13 +629,14 @@ def summary(
         Return format is either pandas.DataFrame {'wide', 'long'} or xarray.Dataset {'xarray'}.
     round_to : int
         Number of decimals used to round results. Defaults to 2.
-    stat_funcs : None or list
-        A list of functions used to calculate statistics. By default, the mean, standard deviation,
-        simulation standard error, and highest posterior density intervals are included.
+    stat_funcs : dict
```

Comment:
This can be a dict, or an iter right? It looks like if you pass a list or tuple you'll still hit this line

https://github.com/arviz-devs/arviz/pull/583/files#diff-f44bb9f0195d3e0d831b3e10ee0ae059R710

Reply:
Oh yes, dict or sequence